pkgs/development/python-modules/fiblary3/default.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, buildPythonPackage
+, isPy3k
+, fetchPypi
+, jsonpatch
+, netaddr
+, prettytable
+, python-dateutil
+, requests
+, six
+, sphinx
+}:
+
+buildPythonPackage rec {
+  pname = "fiblary3";
+  version = "0.1.8";
+  disabled = !isPy3k;
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "ab666088d1996e1cc510ff91c9ff00ac14c7304d327d478ad948b3ea0c27668e";
+  };
+
+  nativeBuildInputs = [
+    sphinx
```

Comment:
`sphinx` is used to build the documentation. Is it really a build requirement?

Reply:
It's mentioned in [requirements.txt](https://github.com/pbalogh77/fiblary/blob/master/requirements.txt) and the build fails without it.  I don't think it's actually used unfortunately.

Reply:
If it is unused we can patch it out.

Reply:
Fixed in my fork.